docker-compose.yml

```diff
@@ -94,6 +94,25 @@ services:
      - --innodb-buffer-pool-size=128M
      - --wait_timeout=65
 
+  nlp:
+    container_name: nlp
+    restart: unless-stopped
+    env_file:
+      - ./.env
+    environment:
+      AMQP_ENDPOINT: 'amqp://rabbitmq:5672'
```

Comment:
This duplicate needs to be removed (why is it there if we defined it in the .env).